web/pdf_find_bar.js

```diff
@@ -80,6 +81,10 @@ var PDFFindBar = (function PDFFindBarClosure() {
     this.caseSensitive.addEventListener('click', function() {
       self.dispatchEvent('casesensitivitychange');
     });
+
+    this.phraseSearch.addEventListener('click', function() {
```

Comment:
I am not sure that it is needed in scope of this PR. This change will not be correlated with other similar code.